ci/pipeline.yml

```diff
@@ -0,0 +1,48 @@
+jobs:
+  - name: test
+    plan:
+    - get: src
+      trigger: true
+    - get: build-timer
+      trigger: true
+    - task: test
+      file: src/ci/tasks/unit-test.yml
+    - task: integration-test
+      file: src/ci/tasks/integration-test.yml
```

Comment:
I believe there is a race condition with your integration test. In a scenario where multiple commits in a short time period are made to the src repo, this job can potentially have parallel executions in flight. This could result in your int-test stepping on itself since it's using the same topic and subscription across processes. I suggest either having the int-test use a unique sub/topic or else adding "serial: true" to the job to prevent parallel executions.

Reply:
Fixed, I also added in retention limits for the uninteresting logs and interruptible to  the long running `build-releases` step